sky/specs/style-guide.md

```diff
@@ -1,5 +1,4 @@
-Flutter Style Guide
-===================
+# Flutter Style Guide
```

Comment:
Me too, but I don't know if this style supports anything beyond 2nd-level headings, so I decided to switch everything to `#` for consistency sake.